wp-content/themes/core/components/routes/index/Index_Controller.php

```diff
@@ -2,17 +2,38 @@
 
 namespace Tribe\Project\Templates\Components\routes\index;
 
+use Tribe\Project\Blocks\Types\Content_Loop\Content_Loop;
+use Tribe\Project\Taxonomies\Featured\Featured;
 use Tribe\Project\Templates\Components\Abstract_Controller;
+use Tribe\Project\Templates\Components\blocks\content_loop\Content_Loop_Controller;
 use Tribe\Project\Templates\Components\breadcrumbs\Breadcrumbs_Controller;
 use Tribe\Project\Templates\Components\sidebar\Sidebar_Controller;
+use Tribe\Project\Templates\Components\text\Text_Controller;
+use Tribe\Project\Templates\Components\Traits\Post_List_Field_Formatter;
 use Tribe\Project\Templates\Models\Breadcrumb;
+use WP_Query;
 
 class Index_Controller extends Abstract_Controller {
+	
```

Comment:
It will definitely keep it cleaner but I think this will exclude the main blog page, using index.php template, right? 

In tasks says "Page Template for archive.php" but then we can not target the main blog page to add the hero image, title, etc. introduced in the new settings page.

Reply:
Discussed and let's move to the Archive.php approach since it's simpler.